### ISSUE TITLE
Alarm/Enrich for daemon.py always says 'missing configuration'.  Parsing dict key is incorrect.

### ISSUE THREAD

**Issue by necroph0s** [outflanknl/RedELK#186]
When parsing the available modules to load within `daemon.py` the dict key is parsed as 

```
<module 'modules.alarm_backendalarm.module'>

#error 

WARNING - helpers - helpers.py - module_should_run -- Missing configuration for enrichment module [<module 'modules.enrich_csbeacon.module' from '/usr/share/redelk/bin/modules/enrich_csbeacon/module.py'>]. Will not run!
```

This causes `daemon.py` to continually fail when searching `config.json` for module configurations.  Obviously, the dict key should be set to `alarm_backend` or the actual module name.  I noticed the variable `module` was overwritten by `importlib.import_module` causing the name of module to be overwritten with the declaration of the module instead.

Here's an example!

![image](https://user-images.githubusercontent.com/22455009/125011634-d4530b00-e036-11eb-8af7-5b542a9adfd9.png)

Potential fix for this includes creating a separate variable and for the module folders name  and using that as the dict key. 

![image](https://user-images.githubusercontent.com/22455009/125011956-5ba07e80-e037-11eb-8461-d877a9b96021.png)

**Comment by necroph0s** [comment]
Not sure if you'd prefer me to do a PR.  Or if what I'm seeing shouldn't be the case.  Not sure - please advise.

**Comment by fastlorenzo** [comment]
Nice catch, thanks for the feedback!

I'll fix it as part of #182
